docs/pages/database-access/guides/dynamic-registration.mdx

```diff
@@ -15,11 +15,11 @@ description: Register/unregister databases without restarting Teleport.
 
 Dynamic database registration allows Teleport administrators to register new
 databases (or update/unregister existing ones) without having to update the
-static configuration and restarting the database agents.
+static configuration and restarting the Teleport Database Services.
 
 To enable dynamic registration, include a `resources` section in your database
-agent configuration with a list of resource label selectors you'd like this
-agent to monitor:
+service configuration with a list of resource label selectors you'd like this
+service to monitor for registering:
```

Comment:
"database service" -> "Database Service"

We should be explicit that we're referring to the Teleport Database Service, since "database service" is otherwise pretty generic

Reply:
updated this and included other suggestions @ptgott